app/assets/javascripts/Components/__tests__/submission_file_upload_modal.test.jsx

```diff
@@ -196,6 +196,16 @@ describe("For the SubmissionFileUploadModal", () => {
         "file2.txt"
       );
     });
+
+    it("should call onSubmit prop with correct arguments when rename input is left blank", () => {
```

Comment:
Great! Since you used `.trim` above (which is good), please add another test case for when the rename input consists only of whitespace.